test/helpers.js

```diff
@@ -0,0 +1,76 @@
+import test from 'ava';
```

Comment:
I've been thinking about this file and it seems unnecessary to me. Every function from `helpers` are used during the main tests, so it doesn't increase coverage.